Library/Homebrew/unpack_strategy/dmg.rb

```diff
@@ -87,8 +87,27 @@ def eject(verbose: false)
         return unless path.exist?
 
         if tries > 1
+          disk_info = system_command(
+            "diskutil",
+            args:         ["info", "-plist", path],
+            print_stderr: false,
+            verbose:      verbose,
+          )
+
+          # For HFS, just use <mount-path>
+          # For APFS, find the <physical-store> corresponding to <mount-path>
+          eject_path = if disk_info.success? && disk_info.plist.respond_to?(:fetch)
```

Comment:
```suggestion
          eject_path = if disk_info.success?
```

If the command succeeded, it should always be a `dict`.

Reply:
Okay. Updated without dict check.
I had originally based on `hdiutil ... -plist` in same file.